src/main/java/top/jsoft/homework/design/builder/Builder.java

```diff
@@ -0,0 +1,12 @@
+package top.jsoft.homework.design.builder;
+
+/**
+ * Created by psygrammator
+ * group jsoft.top
+ */
+public interface Builder {
```

Comment:
Please, avoid Builder interfaces, as usual it should be used as internal incapsulated implementation